### PR TITLE
Mailchimp

### PR DESCRIPTION
**Pull request by joel-molero**

### CODE REVIEW

File: staff/joel-molero/mailchimp/styles.sass [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    display: flex
+    justify-content: space-around
+    flex-direction: row
+    height: 300px
+
+.footerp
+    color: white
+    text-align: center
+    padding-bottom: 30px
+    font-family: 'Work Sans', sans-serif
```
Comment (seand52) [inline staff/joel-molero/mailchimp/styles.sass:85]: I noticed you mentioned the same font-family in multiple places. Maybe you can declare it in a more general selector? or store it as a variable?

File: staff/joel-molero/mailchimp/index.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mailchimp</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="topbar">
+            <div class="mailchimplogo">
+                <p><img src="../mailchimp/images/img/main_logo.png" alt=""></p>
```
Comment (Sergioluz90) [inline staff/joel-molero/mailchimp/index.html:14]: -Alt should not be empty

-If you add to img a Width=100%; attribute in css you will have an adaptive logo

File: staff/joel-molero/mailchimp/index.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+            </div>
+            <div class="button">
+                <button class="signupfree">Sign Up Free</button>
```
Comment (Sergioluz90) [inline staff/joel-molero/mailchimp/index.html:17]: To have an adaptive button:

    font-size: 1.2vw;
    display: inline;
    white-space: nowrap;

File: staff/joel-molero/mailchimp/index.html [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+            </div>
+            <div class="mainnav">
+                <nav class="topnav">
+                    <ul>
+                        <li class="li"><a class="link" href="fakeurl" id="pricing">Pricing</a></li>
```
Comment (Sergioluz90) [inline staff/joel-molero/mailchimp/index.html:22]: font-size: 1.2vw; (the same as button "free sign up")

File: staff/joel-molero/mailchimp/index.html [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+                        <li class="li3"><form id="search">
+                            <input type="text" placeholder="Search"></form>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+        <div class="title-img">
+                <div class="img">
+                    <img class="monkey" src="./images/easy-email-newsletters-banner-png8.png" alt="chimp">
```
Comment (Sergioluz90) [inline staff/joel-molero/mailchimp/index.html:36]: To have it adaptive:
> height:81%;
> padding-left:40%
>

File: staff/joel-molero/mailchimp/index.html [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+                </div>
+                <div class="title">
+                    <h1 class="maintitle">Easy Email Newsletters</h1>
```
Comment (Sergioluz90) [inline staff/joel-molero/mailchimp/index.html:39]: > font-size: 5vw;

File: staff/joel-molero/mailchimp/index.html (hunk 5 above)
Comment (joel-molero) [reply staff/joel-molero/mailchimp/index.html:36]: That kinda messes up the image and the whole style of the front page, plus the bottom section is not responsive either, so it just overlays with the text. Might be a good idea to implement on future versions though.

File: staff/joel-molero/mailchimp/styles.sass (hunk 1 above)
Comment (joel-molero) [reply staff/joel-molero/mailchimp/styles.sass:85]: I like it this way to keep it more visual for me and easier to manipulate.

**Pull request merged**